src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const element = document.getElementsByClassName('population');
+let allPopulationNumber = 0;
+
+for (const key of element) {
+  allPopulationNumber += +key.innerHTML.replaceAll(',', '');
+}
+
+const averagePopulation = allPopulationNumber / element.length;
+
+const classTotalPopulation = document.querySelector('.total-population');
+
+classTotalPopulation.innerHTML = allPopulationNumber;
+
+const classAveragePopulation = document.querySelector('.average-population');
+
+classAveragePopulation.innerHTML = averagePopulation;
```

Comment:
The average population number should be formatted with a thousands separator before being assigned to the innerHTML of classAveragePopulation, as required by the task description.